source/index.rst

```diff
@@ -65,6 +65,7 @@ These are institutions who were early adopters or provided HPC resources for dev
   how-tos/debug
   enable-desktops
   how-tos/app-development
+  how-tos/support_ticket
```

Comment:
I don't think this should be a top level menu item.

Reply:
OK. Will move into one of the existing sections

Reply:
Moved under `Configuration Reference`